plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1000,9 +1000,9 @@ def _get_gas_price_strategy(
             return None
 
         _default_logger.debug(f"Using strategy: {gas_price_strategy}")
-        gas_price_strategy_getter = cast(
-            Callable, self._gas_price_strategy_callables.get(gas_price_strategy)
-        )
```

Comment:
We actually perform a check if the `gas_price_strategy` is set correctly 

https://github.com/valory-xyz/open-aea/blob/563aa212dff256eacd59b7299e428a86e4ed7a54/plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py#L996-L1000